main.js

```diff
@@ -0,0 +1,6 @@
+let Question = function(question, answer1, answer2, correctAnswer) {
+    this.question = question; 
+    this.answer1 = answer1; 
+    this.answer2 = answer2; 
```

Comment:
Good job jerick on creating this. But I think this part is redundant so it should be removed.